### PR TITLE
task: Refine interactive bots task.

### PR DESCRIPTION
**Pull request by trueskawka**
Implemented most of @timabbotts' comments and made sure the file follows the template.

### CODE REVIEW

**Comment by timabbott** [comment]
Also, you should add a script to actually upload these tasks.

File: tasks/interactive-bots.md [hunk 1]
```diff
@@ -128,28 +151,38 @@ Follow the tutorial below to create your first simple bot that sends private mes
...
+Let *feature* be the feature mentioned in the task that brought you here.
 
-Some ideas on what the bots could do:
+1. Make sure you understand what is expected of the feature - what messages should the bot react to, what should it do with the message content and where should it post any output.
 
-  - post to an issue tracker like GitHub
+2. Create your interactive bot at `~/zulip/contrib_bots/lib` as a `.py` file. Make sure it fulfills the requirements.
 
-  - post events to a calendar like Google Calendar
+3. Make sure all the tests pass, using `tools/test-all`.
```
Comment (trueskawka) [inline tasks/interactive-bots.md]: What should we recommend for testing here? 

I'd think encouraging students to test is generally good practice and sometimes changes they make might affect tests unexpectedly (e.g. one student was changing lines not needed for his feature and was thus failing linter tests).

**Comment by trueskawka** [comment]
@timabbott Added the script for uploading and refined the tasks.

What tests should the students run for the bots?

**Comment by timabbott** [comment]
The bots don't really have a test suite yet, so I think we should probably just ask them to document how to use their bot, and then the mentor can verify it works.  We'll need to figure out a test suite strategy for them later.

**Comment by timabbott** [comment]
Merged!  We can do revisions on top of this.

**Pull request closed without merging**